Packages/com.llealloo.audiolink/Runtime/Scripts/AudioLink.cs

```diff
@@ -732,17 +732,25 @@ public override void OnDeserialization()
         }
 #endif
 
+        private const int MaxLength = 32;
+        private int[] codePoints = new int[MaxLength];
+        public const int VecsLength = MaxLength / 4;
+        private Vector4[] vecs = new Vector4[VecsLength];
```

Comment:
Since this is now in class scope, maybe rename it to something more telling, like packedGlobalTextVecs

Reply:
Along with the name of the length